src/test/resources/TestNG.xml

```diff
@@ -6,13 +6,16 @@
     <test name="Example Test" preserve-order="false">
         <classes>
             <class name="Homework17">
-                <methods>
-                    <include name="addSongToPlaylist" />
-                </methods>
+<!--                <methods>-->
+<!--                    <include name="addSongToPlaylist" />-->
+<!--                </methods>-->
+                <class name="BaseTest">
+                <class name="LoginTests">
+                <class name="ChangeUserNameProfile">
+                <class name="PlaylistTests">
             </class>
         </classes>
     </test>
 </suite>
 
-<!--<class name="RegistrationTests"/>-->
-<!--<class name="ChangeUsernameTests"/>-->
+
```

Comment:
To run tests, I fixed


```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

<suite name="TestNG">

  <test name="Example Test" preserve-order="false">
    <classes>
      <class name="LoginTests"/>
      <class name="BaseTest"/>
      <class name="Homework17"/>
      <class name="ChangeUserNameProfile"/>
      <class name="PlaylistTests"/>
      </classes>
  </test>
</suite>
```

![image](https://user-images.githubusercontent.com/104761482/198947432-d3826b35-8fc7-4427-ae70-141f47e1f30c.png)